skiko/src/commonMain/kotlin/org/jetbrains/skiko/GraphicsApi.kt

```diff
@@ -1,7 +1,7 @@
 package org.jetbrains.skiko
 
 enum class GraphicsApi {
-    UNKNOWN, SOFTWARE, DIRECT_SOFTWARE, OPENGL, DIRECT3D, VULKAN, METAL, WEBGL
+    UNKNOWN, SOFTWARE, AWT_SOFTWARE, OPENGL, DIRECT3D, VULKAN, METAL, WEBGL
```

Comment:
Not sure I agree on naming, and even worse changing semantics of the same API. Previous one looked better.

Reply:
Discussed offline, let's name them `FAST_SOFTWARE` and `COMPAT_SOFTWARE`.

Reply:
Fixed. The names are SOFTWARE_FAST and SOFTWARE_COMPAT